api/src/main/java/org/pmiops/workbench/utils/mappers/LeonardoMapper.java

```diff
@@ -59,27 +72,55 @@ ListRuntimeResponse toApiListRuntimeResponse(
   @Mapping(target = "dataprocConfig", ignore = true)
   Runtime toApiRuntime(LeonardoGetRuntimeResponse runtime);
 
+  @Mapping(target = "createdDate", source = "auditInfo.createdDate")
+  @Mapping(target = "autopauseThreshold", ignore = true)
+  @Mapping(target = "toolDockerImage", ignore = true)
```

Comment:
What's the status on this? Is it intentionally omitted by Leo?

Reply:
I verified that it is not actually sent through in the JSON response. Just shot off a message to the Leo team about it.